cmake_include/FindMXML.cmake

```diff
@@ -44,8 +44,4 @@ if(NOT HAVE_MXML)
     message(FATAL_ERROR "User requested MXML not found!")
 endif()
 
-if(CMAKE_HOST_UNIX)
-    list(APPEND NAPI_LINK_LIBS mxml)
-else()
-    list(APPEND NAPI_LINK_LIBS mxml mxml1)
-endif()
+list(APPEND NAPI_LINK_LIBS ${MXML_LIBFILES})
```

Comment:
I will give this a try and if it works for you we can keep it for the moment. I was thinking about a better solution anyhow (for all the libraries).